cmd/snap-update-ns/common_test.go

```diff
@@ -24,6 +24,7 @@ import (
 	"os"
 	"path/filepath"
 
+	"golang.org/x/net/context"
```

Comment:
This is probably automatically added by your editor. Please use `context` from the standard library.

Reply:
Thanks for catching this!